csharp/Facebook.Yoga/Facebook.Yoga.csproj

```diff
@@ -0,0 +1,135 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
This is a NETStandard 1.1 PCL project that allows us to build one DLL that targets .NET4.5 and above...